blackbox/hdfs/src/tests.py

```diff
@@ -100,26 +102,48 @@ def _get_hadoop(self):
                             'hadoop-{version}'.format(version=HADOOP_VERSION))
 
     def setUp(self):
-        cmd = ['bash',
-               self.hadoop_bin, 'jar',
-               self.hadoop_mapreduce_client, 'minicluster',
-               '-nnport', NN_PORT, '-nomr', '-D', 'dfs.replication=0']
+        cmd = [
+            self.hadoop_bin, 'jar',
+            self.hadoop_mapreduce_client, 'minicluster',
+            '-nnport', NN_PORT, '-nomr', '-format',
+            '-D', 'dfs.replication=0',
+            '-D', 'dfs.client.use.datanode.hostname=true',
+            '-D', 'dfs.datanode.use.datanode.hostname=true',
+        ]
 
         JAVA_HOME = os.environ.get('JAVA_HOME', '/usr/lib/jvm/java-8-openjdk/')
+
+        print('')
+        print('>>> ' + 'JAVA_HOME=' + JAVA_HOME)
```

Comment:
I can remove them. Guess they are not *that* interesting any more once it's stable.